modules/get-started/pages/licensing/overview.adoc

```diff
@@ -10,6 +10,12 @@ Redpanda, Redpanda Console, and Redpanda Connect are available in community and
 ** https://github.com/redpanda-data/connect[Redpanda Connect^]
 * Redpanda Enterprise Edition requires a license key and includes additional features.
 
+.Ready to try Redpanda Enterprise Edition?
```

Comment:
@JakeSCahill: you could do the same thing here: 
:tip-caption: Ready to go into production?

[TIP]